lib/cylc/gui/SuiteControlTree.py

```diff
@@ -238,6 +238,18 @@ def _set_tooltip( self, widget, tip_text ):
         tip.enable()
         tip.set_tip( widget, tip_text )
 
+    def _set_cell_text_time(self, column, cell, model, iter_, n):
+        """Remove the date part if it matches the last update date."""
+        if "T" in self.updater.dt:
```

Comment:
The two if's can be combined, and you still need a date_time_string when last_update_date is None